src/routes/gallery/components/imageGallery/ImageModal.svelte

```diff
@@ -1,10 +1,9 @@
 <script lang="ts">
   import { createEventDispatcher, onDestroy, onMount } from 'svelte'
 
-  import { deleteImageFromWNFS } from '$lib/gallery'
-  import { galleryStore } from '../../../stores'
-  import type { Gallery, Image } from '$lib/gallery'
-  import { fissionServerUrl } from '$lib/app-info'
+  import { fissionServerUrl } from '$lib/app-info';
```

Comment:
It just occurred to me that a better name for this variable could be `ipfsGatewayUrl`.

Reply:
yes please! Some of this is "fission server" -- e.g. endpoints of our server / usernames / etc at `runfission.com`. But also this SHOULD be able to be loaded by any IPFS Gateway.

including things like e.g. uploading / persisting via the Web3Storage SDK, but then serving up through a custom Fission domain.

Reply:
good call, i'll update it!